sql/python/sqlflow_submitter/db_writer/hive.py

```diff
@@ -11,22 +11,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
 from .base import BufferedDBWriter
+import tempfile
+import subprocess
 
 class HiveDBWriter(BufferedDBWriter):
-    def __init__(self, conn, table_name, table_schema, buff_size):
-        return super().__init__(conn, table_name, table_schema, buff_size)
+    def __init__(self, conn, table_name, table_schema, buff_size=10000):
+        super().__init__(conn, table_name, table_schema, buff_size)
 
     def flush(self):
-        statement = '''insert into table {} ({}) values({})'''.format(
-            self.table_name,
-            ", ".join(self.table_schema),
-            ", ".join(["%s"] * len(self.table_schema))
-        )
-        cursor = self.conn.cursor()
-        try:
-            cursor.executemany(statement, self.rows)
-            self.conn.commit()
-        finally:
-            cursor.close()
-            self.rows = []
+        tmp_f = tempfile.NamedTemporaryFile()
+        with open(tmp_f.name, "w") as f:
+            for row in self.rows:
```

Comment:
seems this is to write rows in the buffer, is there a way to write all the rows before writing to hive?